src/main/java/com/weaverplatform/service/controllers/StoreController.java

```diff
@@ -0,0 +1,124 @@
+package com.weaverplatform.service.controllers;
+
+
+import com.weaverplatform.sdk.Weaver;
+import com.weaverplatform.service.payloads.FileFromStoreRequest;
+import com.weaverplatform.service.util.Props;
+import com.weaverplatform.service.util.ZipWriter;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+public class StoreController {
+
+
+  public static final String WEAVER_URI = Props.get("WEAVER_URI", "weaver.uri");
+  public static final String USER = Props.get("SERVICE_USER","service.user");
+  public static final String PASSWORD = Props.get("SERVICE_PASSWORD", "service.password");
+
+  public static Weaver getWeaver(String project) {
+    Weaver instance = new Weaver();
+    instance.setUri(WEAVER_URI);
+    instance.setUsername(USER);
+    instance.setPassword(PASSWORD);
+    instance.login();
+    instance.setProject(project);
+    return instance;
+  }
+
+  public static Route fileFromStore = (Request request, Response response) -> {
+
+    String project = request.queryParamOrDefault("project", null);
+    if(project == null) {
+      response.status(500);
+      response.type("application/json");
+      return "{\"success\":false,\"message\":\"Please provide project\"}";
+    }
+
+    String zipKey = request.queryParamOrDefault("zipKey", null);
+    if(zipKey == null) {
+      response.status(500);
+      response.type("application/json");
+      return "{\"success\":false,\"message\":\"Please provide zipKey\"}";
+    }
+
+    String path = request.queryParamOrDefault("path", null);
+    if(path == null) {
+      response.status(500);
+      response.type("application/json");
+      return "{\"success\":false,\"message\":\"Please provide path\"}";
+    }
+
+    String fileId = request.queryParamOrDefault("fileId", null);
+    if(fileId == null) {
+      response.status(500);
+      response.type("application/json");
+      return "{\"success\":false,\"message\":\"Please provide fileId\"}";
```

Comment:
Given the amount of literal json strings, wouldn't it just be easier to have a 
```
class Result {
  public boolean success;
  public String message
}
```
And jsonify that?